schema/test/unit/computed_columns/form_change_as_reporting_requirement_test.sql

```diff
@@ -48,7 +48,7 @@ select is(
     ) select pg_typeof(cif.form_change_as_reporting_requirement((select * from record)))::text
   ),
   (
-    'cif.reporting_requirement'::text
+    'reporting_requirement'::text
```

Comment:
I'm confused why this started failing, but a possible solution could be to not cast the pg_typeof result to text and cast 'reporting_requirement' to regtype.